src/Wellcome.Dds/Wellcome.Dds.Server/Controllers/BornDigitalExtensionsController.cs

```diff
@@ -0,0 +1,100 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.FeatureManagement.Mvc;
+using SixLabors.ImageSharp;
+using SixLabors.ImageSharp.Formats.Png;
+using SixLabors.ImageSharp.PixelFormats;
+using SixLabors.Fonts;
+using SixLabors.ImageSharp.Drawing.Processing;
+using SixLabors.ImageSharp.Processing;
+using Utils;
+using Utils.Web;
+
+namespace Wellcome.Dds.Server.Controllers;
+
+[FeatureGate(FeatureFlags.PresentationServices)]
+[Route("extensions/born-digital")]
+public class BornDigitalExtensionsController : ControllerBase
+{
+    private string WebRootPath { get; set; }
+    
+    public BornDigitalExtensionsController(IWebHostEnvironment environment)
+    {
+        WebRootPath = environment.WebRootPath;
+    }
+    
+    [HttpGet("placeholder-canvas/{*pathParts}")]
+    public async Task<IActionResult> PlaceholderCanvas(string pathParts)
+    {
+        var parts = new PlaceholderParts(pathParts);
+        using Image img = new Image<Rgba32>(1000, 800);
+        FontCollection collection = new();
+        FontFamily family = collection.Add(GetFontPath());
+        Font largeFont = family.CreateFont(32, FontStyle.Regular);
+        Font smallFont = family.CreateFont(16, FontStyle.Regular);
+        img.Mutate(x=> x.DrawText(
+            $"File type: {parts.MimeType}", 
+            largeFont, Color.Black, new PointF(20, 20)));
+        img.Mutate(x=> x.DrawText(
+            $"PRONOM key: {parts.PromomKey}", 
+            largeFont, Color.Black, new PointF(20, 80)));
+        img.Mutate(x=> x.DrawText(
+            $"This is a placeholder canvas.", 
+            smallFont, Color.Black, new PointF(20, 140)));
+        img.Mutate(x=> x.DrawText(
+            $"The original file is available in the rendering property of this canvas.", 
+            smallFont, Color.Black, new PointF(20, 165)));
+        
+        Response.ContentType = "image/png";
+        Response.CacheForDays(28);
+        await img.SaveAsync(Response.Body, new PngEncoder());
+        return new EmptyResult();
```

Comment:
At a quick scan it looks like this doesn't return anything due to the `EmptyResult`. Unsure of perf reduction but could it be something mroe obvious?

```cs
var ms = new MemoryStream();
await img.SaveAsync(ms, new PngEncoder());
return File(ms, "image/png");
```

Or a custom `ImageResult()`.